<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddManga</class>
 <widget class="QDialog" name="DialogAddManga">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add manga</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/DBA_Icon.png</normaloff>:/images/DBA_Icon.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="HLay_0">
     <item>
      <layout class="QVBoxLayout" name="VLay_0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="HLay_1">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Manga</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="HSpacer_0">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="CheckBox_LookLater">
           <property name="whatsThis">
            <string>If you want to read this manga later - deliver a checkbox for display it in the special filter</string>
           </property>
           <property name="text">
            <string>Want to read</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="CheckBox_Editing">
           <property name="whatsThis">
            <string>If you didn't finish editing - deliver a checkbox for display it in the special filter</string>
           </property>
           <property name="text">
            <string>Editing</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="TBtn_Search">
           <property name="toolTip">
            <string>Search on shikimori.org</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LineEdit_Title">
           <property name="maxLength">
            <number>128</number>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Title</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="CBox_Year">
           <property name="whatsThis">
            <string>If this checkbox isn't active - year won't be displayed</string>
           </property>
           <property name="text">
            <string>Year:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="SpinBox_Year">
           <property name="whatsThis">
            <string>You can use a mouse wheel for value change</string>
           </property>
           <property name="minimum">
            <number>1900</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="value">
            <number>2000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="VLay_AltTitle">
         <property name="topMargin">
          <number>0</number>
         </property>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="HLay_AuthorAndSound">
         <property name="topMargin">
          <number>0</number>
         </property>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="TabWidget_Series">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>120</height>
          </size>
         </property>
         <property name="whatsThis">
          <string>You can use a mouse wheel for switching on tabs</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_allSeries">
          <attribute name="title">
           <string>Total books</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="Lbl_aVol">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>&amp;Volumes</string>
             </property>
             <property name="buddy">
              <cstring>SpinBox_aVol</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="Lbl_aCh">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>&amp;Charapters</string>
             </property>
             <property name="buddy">
              <cstring>SpinBox_aCh</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="Lbl_aPages">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>&amp;Pages</string>
             </property>
             <property name="buddy">
              <cstring>SpinBox_aPages</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QSpinBox" name="SpinBox_aVol">
             <property name="whatsThis">
              <string>You can use a mouse wheel for value change</string>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="SpinBox_aCh">
             <property name="whatsThis">
              <string>You can use a mouse wheel for value change</string>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSpinBox" name="SpinBox_aPages">
             <property name="whatsThis">
              <string>You can use a mouse wheel for value change</string>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_lookSeries">
          <attribute name="title">
           <string>Read</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="Lbl_vVol">
             <property name="text">
              <string>&amp;Volume</string>
             </property>
             <property name="buddy">
              <cstring>SpinBox_vVol</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="Lbl_vCh">
             <property name="text">
              <string>&amp;Charapter</string>
             </property>
             <property name="buddy">
              <cstring>SpinBox_vCh</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="Lbl_vPage">
             <property name="text">
              <string>&amp;Page</string>
             </property>
             <property name="buddy">
              <cstring>SpinBox_vPages</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QSpinBox" name="SpinBox_vVol">
             <property name="whatsThis">
              <string>You can use a mouse wheel for value change</string>
             </property>
             <property name="maximum">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="SpinBox_vCh">
             <property name="whatsThis">
              <string>You can use a mouse wheel for value change</string>
             </property>
             <property name="maximum">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSpinBox" name="SpinBox_vPages">
             <property name="whatsThis">
              <string>You can use a mouse wheel for value change</string>
             </property>
             <property name="maximum">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="TabWidget_Info">
         <property name="whatsThis">
          <string>You can use a mouse wheel for switching on tabs</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Genres</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="LineEdit_Tags">
             <property name="text">
              <string/>
             </property>
             <property name="maxLength">
              <number>256</number>
             </property>
             <property name="dragEnabled">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>Enter through a comma, or choose from the list lower</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="ListView_Tags">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustToContents</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::MultiSelection</enum>
             </property>
             <property name="isWrapping" stdset="0">
              <bool>true</bool>
             </property>
             <property name="resizeMode">
              <enum>QListView::Adjust</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Description</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPlainTextEdit" name="PlainTextEdit_Description">
             <property name="placeholderText">
              <string>Description</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Media</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="HLayout_Dir">
             <item>
              <widget class="QLineEdit" name="LineEdit_Dir">
               <property name="maxLength">
                <number>256</number>
               </property>
               <property name="dragEnabled">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string>Directory</string>
               </property>
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="TBtn_ChooseDir">
               <property name="toolTip">
                <string/>
               </property>
               <property name="text">
                <string>*</string>
               </property>
               <property name="icon">
                <iconset resource="resource.qrc">
                 <normaloff>:/images/folder.png</normaloff>:/images/folder.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLineEdit" name="LineEdit_URL">
             <property name="maxLength">
              <number>256</number>
             </property>
             <property name="dragEnabled">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>URL</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="VSpacer_0">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>138</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="ImageCover" name="Lbl_ImageCover">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>3</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>540</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>275</width>
         <height>825</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Drop image here, or a double click for a file choice</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/images/DropHere.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="BtnBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageCover</class>
   <extends>QLabel</extends>
   <header location="global">widgets/imagecover.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>LineEdit_Title</sender>
   <signal>returnPressed()</signal>
   <receiver>TBtn_Search</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
